helm-charts/sonatype-nexus-repository/templates/statefulset.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Chart.Name }}-{{ .Chart.Version  | replace "." "-"}}-{{ .Values.statefulset.name }}
+  namespace: {{ .Values.namespaces.nexusNs }}
+  labels:
+    app: nxrm
+spec:
+  replicas: {{.Values.statefulset.replicaCount}}
+  serviceName: "{{ .Chart.Name }}-{{ .Chart.Version  | replace "." "-"}}-{{ .Release.Name }}-{{ .Values.statefulset.name }}"
+  selector:
+    matchLabels:
+      app: nxrm
+  template:
+    metadata:
+      labels:
+        app: nxrm
+    spec:
+      initContainers:
+        - name: chown-nexusdata-owner-to-nexus
+          image: {{ .Values.statefulset.initContainer.image.repository }}:{{ .Values.statefulset.initContainer.image.tag }}
+          resources:
+            {{ toYaml .Values.statefulset.initContainer.resources | nindent 12 }}
+          command: [/bin/sh]
+          args:
+            - -c
+            - >-
+              chown -R '200:200' /nexus-data
+          volumeMounts:
+            - name: nexusdata
+              mountPath: /nexus-data
+      terminationGracePeriodSeconds: {{ .Values.statefulset.container.terminationGracePeriod }}
+      containers:
+        - name: nxrm-app
+          image: {{ .Values.statefulset.container.image.repository }}:{{ .Values.statefulset.container.image.tag }}
+          # See help documentation, these are minimum system requirements
+          resources:
+            {{ toYaml .Values.statefulset.container.resources | nindent 12 }}
+          securityContext:
+            runAsUser: 200
+          imagePullPolicy: {{ .Values.statefulset.container.pullPolicy }}
+          ports:
+            - containerPort: {{ .Values.statefulset.container.containerPort }}
+          env:
+            - name: DATASTORE_CLUSTERED_ENABLED
```

Comment:
as discussed add `IS_OPEN_SHIFT_DEPLOYMENT` to env

Reply:
Added